### PR TITLE
Moving common code to a central repository

### PR DESCRIPTION
**Pull request by sshniro**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## [Unreleased]
 ### Added
 - Allow to configure the title of the issue raised, using the parameter `issue_title`. [#10](https://github.com/zaproxy/action-baseline/issues/10)
+- Moved the common code to action-common-scans repository
```
Comment (thc202) [inline CHANGELOG.md:9]: This could be added under the Changed section.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 ### Added
-- Allow to configure the title of the issue raised, using the parameter `issue_title`. [#10](https://github.com/zaproxy/action-baseline/issues/10)
+- Allow to configure the title of the issue raised, using the parameter `issue_title`. [#10](https://github.com/zaproxy/action-baseline/issues/10
+
+### Changed
+- Moved the common code to action-common-scans repository
```
Comment (thc202) [inline CHANGELOG.md:11]: `actions-common repository`.

File: README.md [hunk 3]
```diff
@@ -26,6 +27,7 @@ configure the parameter to use the weekly builds.
 
 **Optional** You can also specify a relative path to the rules file to ignore any alerts from the ZAP scan. Make sure to create
 the rules file inside the relevant repository. The following shows a sample rules file configuration.
+Make sure to use the checkout action(`actions/checkout@v2`) for the repository to download the zap rules files to the ZAP Action.
```
Comment (thc202) [inline README.md:30]: `Make sure to checkout the repository (actions/checkout@v2) to provide the ZAP rules to the scan action.`?

File: action.yml [hunk 4]
```diff
@@ -21,7 +21,7 @@ inputs:
     description: 'Additional command line options'
     required: false
   issue_title:
-    description: 'The title for the GitHub issue that is created'
+    description: 'The title for the GitHub issue to be created'
```
Comment (thc202) [inline action.yml:24]: If we change here we should change in the README too.

File: CHANGELOG.md (hunk 2 above)
Comment (sshniro) [reply CHANGELOG.md]: done, added the repo as well

File: README.md (hunk 3 above)
Comment (sshniro) [reply README.md]: replaced.

File: action.yml (hunk 4 above)
Comment (sshniro) [reply action.yml:24]: added in the readme

**Comment by thc202** [comment]
Thank you!

**Pull request merged**